src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double width;
+
+    public Rectangle(String color, double length, double width) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+
+    @Override
+    public double area() {
```

Comment:
Method names should be verbs or verb phrases to describe the action they perform. Consider renaming the 'area' method to something like 'calculateArea' to clearly indicate its purpose.